### DISCUSSION TITLE
Power strip with individual power monitoring per socket

### DISCUSSION THREAD

**Discussion by scasic** [arendst/Tasmota#14365] [Q&A] [answered]
Dear all,

i'm looking for a multi socket power strip with individual power monitoring per each socket. At the moment I found the only one which has such feature - Kasa HS300. But there is only 120V version and it is not supported by Tasmota.
Maybe there is a known 220V alternative flashable with Tasmota?

**Comment by barbudor** [comment, marked as answer]
All strips we are aware are listed in the Template Repository : https://templates.blakadder.com/
I've never found any of such strip.
One option, not very compact and elegant I would admit, is to build one yourself using devices such as Sonoff Dual POW R3 or Shelly 2.5

**Reply by sfromis** [reply]
The Shelly 2.5 is compact enough to integrate within a strip, if you block off every third socket, and to a bit of chiseling. Obviously be very careful about doing safe wiring, and allow air circulation for cooling.

You could also do something with a PZEM and CT clamps at suitable places within.
